devops/dockerfiles/node-new/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# syntax=docker/dockerfile:1
+FROM paritytech/ci-linux:production AS base
+RUN cargo install cargo-chef
+WORKDIR /app
+
+FROM base AS planner
+COPY . .
+RUN cargo chef prepare
+
+FROM base AS builder
+COPY --from=planner /app/recipe.json recipe.json
+# Build dependencies - this is the caching Docker layer!
+RUN cargo chef cook --check --release --no-default-features -p mangata-node
+
+# Build application
+COPY . .
+RUN cargo build --release -p mangata-node --no-default-features --features=mangata-rococo,mangata-kusama \
+  && mv /app/target/release/mangata-node /app/node \ 
```

Comment:
which one is the output of /release/mangata-node?  kusama or rococo?

Reply:
that seems to be true at the moment, right @iStrike7 , do you see any problem with that ?